microsoft/vscode
# Add #96264: Checkout git with detach flag

This PR fixes  #96264

### Description
This PR allows the user to checkout from a remote branch with a detached head. It will add `config.checkoutDetached` as a setting to the default git extension. If this option is set to true, the `--detach` flag will be added to the git checkout command. 

### How to Test
To test this feature you'll have to checkout a remote branch from the git menu. If the setting is enabled the new branch should have a detached head.
